src/extensions/default/AutoUpdate/main.js

```diff
@@ -59,6 +60,28 @@ define(function (require, exports, module) {
     var MAX_DOWNLOAD_ATTEMPTS = 6,
         downloadAttemptsRemaining;
 
+    var AUTOUPDATE_DOWNLOAD_START = "DownloadStarted";
+    var AUTOUPDATE_DOWNLOAD_COMPLETED = "DownloadCompleted";
+    var AUTOUPDATE_DOWNLOAD_FAILED = "DownloadFailed";
+    var AUTOUPDATE_DOWNLOAD_COMPLETE_USER_CLICK_RESTART = "DownloadCompletedAndUserClickedRestart";
+    var AUTOUPDATE_DOWNLOAD_COMPLETE_USER_CLICK_LATER = "DownloadCompletedAndUserClickedLater";
+    var AUTOUPDATE_DOWNLOADCOMPLETE_UPDATE_BAR_RENDERED = "DownloadCompleteUpdateBarRendered";
+    var AUTOUPDATE_INSTALLATION_FAILED = "installationFailed";
+    var AUTOUPDATE_INSTALLATION_SUCCESS = "installationSuccess ";
```

Comment:
@niteskum , a better way of writing this would be to club the log names into a single enum. Also, please follow the same case/notation while defining log names.

Reply:
Use camelCase for all strings